backend/controllers/cellMove/confirmCellMove.ts

```diff
@@ -85,8 +85,12 @@ export default ({ prisonApi, whereaboutsApi }) => {
     return res.redirect(`/prisoner/${offenderNo}/cell-move/confirmation?cellId=${cellId}`)
   }
 
-  const makeCSwap = async (res, { bookingId, agencyId, offenderNo }) => {
-    await prisonApi.moveToCellSwap(res.locals, { bookingId })
+  const makeCSwap = async (req, res, { bookingId, agencyId, offenderNo }) => {
+    const { username } = req.session.userDetails
+
+    const systemContext = await systemOauthClient.getClientCredentialsTokens(username)
+
+    await cellAllocationApi.moveToCellSwap(systemContext, { bookingId })
```

Comment:
This is the same as the call that was being made to prisonApi apart from the use of systemContext rather than user context